src/synapse/azext_synapse/constant.py

```diff
@@ -0,0 +1,34 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+from enum import Enum
+
+DOTNET_FILE = "local:///usr/hdp/current/spark2-client/jars/microsoft-spark.jar"
+DOTNET_CLASS = "org.apache.spark.deploy.dotnet.DotnetRunner"
+EXECUTOR_SIZE = {'Small': {'Cores': 4, 'Memory': '28g'}, 'Medium': {'Cores': 8, 'Memory': '56g'}}
+SPARK_DOTNET_ASSEMBLY_SEARCH_PATHS_KEY = 'spark.yarn.appMasterEnv.DOTNET_ASSEMBLY_SEARCH_PATHS'
```

Comment:
Add `Large`?

Reply:
fixed

---
In reply to: [417305884](https://github.com/Azure/azure-cli-extensions/pull/1581#discussion_r417305884) [](ancestors = 417305884)